dashboards/flink/metadata.yaml

```
@@ -0,0 +1,20 @@
+sample_dashboards:
+  -
+    category: flink
```

```suggestion
    category: Flink
```

Can you capitalize Flink here and in the other category field?

---

updated